Model/Config/Source/PrefTariffs.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Reach\Payment\Model\Config\Source;
+
+class PrefTariffs implements \Magento\Framework\Option\ArrayInterface
+{
+
+    /**
+     * Return array of options as value-label pairs, eg. value => label
+     *
+     * @return array
+     */
+    public function toOptionArray()
+    {
+        return [
+            ['value' => 1, 'label' => __('True')],
```

Comment:
since the values are still 1 and 0 won't 1/0 be sent through to DHL? It's good that the labels are set to True/False but my understanding of MAG-65 was that True/False needed to be sent along to DHL in the FLC API request.

This might help: https://magento.stackexchange.com/questions/139148/magento-2-how-to-read-the-values-in-etc-config-xml

Using the code above,

```
$value = $this->scopeConfig->getValue('section/group/value1', \Magento\Store\Model\ScopeInterface::SCOPE_STORE);  

will return 1 and

$value = $this->scopeConfig->isSetFlag('section/group/value1', \Magento\Store\Model\ScopeInterface::SCOPE_STORE);  

will return true.
```

Reply:
Actually there is a piece of code in pre-existing DutyCalculator.php that turns this 1/0 to true, false and as far as DHL is concerned it would get right value. 
DHL call would fail with full commit by Ryan as he changed both config and DutyCalculator.php in a way that DHL would get string `True` or `False` with his committed code as is.